### DISCUSSION TITLE
Help please - power surge and modem problems

### DISCUSSION THREAD

**Discussion by Juggler00** [TD22057/insteon-mqtt#525] [General]
The transformer feeding our house had one of its 120V feeds die yesterday. They've replaced the transformer and I'm now having problems with my Insteon setup.

I use a 2413U modem attached to a Raspberry Pi running insteon-mqtt in a docker container; it's been running for several years. After power was restored to my house, I noticed that my Insteon devices were not sending their commands to the modem. I could control the devices through MQTT (HA), but not vice versa.

Fortunately (I thought), I did have a spare modem and switched them making sure I updated the config file with the new modem address. I was in the process of re-joining/pairing the devices and was successful in updating 1 dimmer. When I issued the command for the second dimmer, I received the following error. Now any command I issue gives me a similar error:
```
root@ac486b6ec58b:/config# insteon-mqtt insteon_mqtt.yaml join 42.72.3f
Commanding dimmer device 42.72.3f (office desk light) cmd=join
Handler timed out 1 of 3 sent: Std: 42.72.3f, Type.DIRECT mh:3 hl:3, 0d 00 
Handler timed out 2 of 3 sent: Std: 42.72.3f, Type.DIRECT mh:3 hl:3, 0d 00 
Handler timed out 3 of 3 sent: Std: 42.72.3f, Type.DIRECT mh:3 hl:3, 0d 00 
ERROR: Handler timed out - no more retries (3 sent)
ERROR: Command timed out
```
Keep in mind this is with a new modem. Can anyone give me any hints as to what I'm seeing here or the cause?

**Comment by f1d094** [comment]
@Juggler00: I just made a [post](https://github.com/TD22057/insteon-mqtt/discussions/530) about this very thing!

Try this: `insteon-mqtt insteon_mqtt.yaml raw-command 42.72.3f 0x1F 0x00`

My problem was with devices that were joined and paired...and I was not having problems with joining or pairing new devices...but I did have several devices timing out when trying to update my scenes. The above command instantly fixed all my errant devices. No idea why.
